x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/ConnectorSyncInfo.java

```diff
@@ -96,17 +96,17 @@ public ConnectorSyncInfo(StreamInput in) throws IOException {
         this.lastSynced = in.readOptionalInstant();
     }
 
-    private static final ParseField LAST_ACCESS_CONTROL_SYNC_ERROR = new ParseField("last_access_control_sync_error");
-    private static final ParseField LAST_ACCESS_CONTROL_SYNC_STATUS_FIELD = new ParseField("last_access_control_sync_status");
-    private static final ParseField LAST_ACCESS_CONTROL_SYNC_SCHEDULED_AT_FIELD = new ParseField("last_access_control_sync_scheduled_at");
-    private static final ParseField LAST_DELETED_DOCUMENT_COUNT_FIELD = new ParseField("last_deleted_document_count");
-    private static final ParseField LAST_INCREMENTAL_SYNC_SCHEDULED_AT_FIELD = new ParseField("last_incremental_sync_scheduled_at");
-    private static final ParseField LAST_INDEXED_DOCUMENT_COUNT_FIELD = new ParseField("last_indexed_document_count");
-    private static final ParseField LAST_SEEN_FIELD = new ParseField("last_seen");
-    private static final ParseField LAST_SYNC_ERROR_FIELD = new ParseField("last_sync_error");
-    private static final ParseField LAST_SYNC_SCHEDULED_AT_FIELD = new ParseField("last_sync_scheduled_at");
-    private static final ParseField LAST_SYNC_STATUS_FIELD = new ParseField("last_sync_status");
-    private static final ParseField LAST_SYNCED_FIELD = new ParseField("last_synced");
+    protected static final ParseField LAST_ACCESS_CONTROL_SYNC_ERROR = new ParseField("last_access_control_sync_error");
```

Comment:
Same question, why the change to protected?

Reply:
In `ConnectorIndexService` I support sorting by:
```
.sort(Connector.ID_FIELD.getPreferredName(), SortOrder.ASC);
``` 
So I need to have access to this field from a different class. 

For `update_x` actions I would need access to most of the parse fields in the connector doc. I decided to change all of the to protected for convenience.

Reply:
But I guess that those fields in Connector are not related to this class, right? We're merely reusing them. I'd say in that case that we copy the fields in the Connector class, as they are not related.